src/main/panel/Sound.java

```diff
@@ -0,0 +1,69 @@
+package main.panel;
+
+
+import java.io.File;
+import java.io.IOException;
+import javax.sound.sampled.*;
+public class Sound {
+    // Nested class for specifying volume
```

Comment:
This is the biggest problem, sound class is not belong to panel type, put it in base will be better